liquidity-module/index.md

```diff
@@ -236,13 +230,22 @@ At your local terminal, enter the IBC module command to transfer your token to t
 Make sure to replace `cosmosxxxxx` with your address, `mytoken` with your token name and `username` with your username.
 
 ```bash
-myblockchaind tx ibc-transfer transfer transfer channel-0 cosmosxxxxx "500mytoken" --from username
+myblockchaind tx ibc-transfer transfer transfer channel-0 cosmosxxxxx "15000000mytoken" --from username
 ```
 
 After your transaction is complete, check your balance on the Gravity DEX testnet to confirm your token transfer.
 
 Make sure to replace `cosmosxxxxx` with your address.
 
+**Tip** Sometimes transactions don't go through in first try. Make sure you check your terminal window created by your relayer. It should have the following output.
```

Comment:
```suggestion
**Tip** Sometimes transactions don't go through on the first try. Make sure you check the terminal window that shows the relayer process and verify that you see output similar to the following output:
```

Reply:
hi @alijnmerchant21 ! which terminal window was "created by your relayer"?

Reply:
I have attached the o/p as a markdown comment